testsuite/tests/kuadrant/defaults_overrides/defaults/test_rules_exclusivity.py

```diff
@@ -0,0 +1,36 @@
+"""Test mutual exclusivity of defaults block and implicit defaults"""
+
+import pytest
+
+from testsuite.policy.rate_limit_policy import Limit
+
+pytestmark = [pytest.mark.kuadrant_only, pytest.mark.limitador]
+
+
+@pytest.fixture(scope="module")
+def authorization(authorization, oidc_provider):
+    """Create AuthPolicy with basic oidc rules inside and outside defaults block"""
+    authorization.identity.add_oidc("inside-defaults", oidc_provider.well_known["issuer"], defaults=True)
+    authorization.identity.add_oidc("outside-defaults", oidc_provider.well_known["issuer"])
+    return authorization
+
+
+@pytest.fixture(scope="module")
+def rate_limit(rate_limit):
+    """Add basic rate limiting rules inside and outside defaults block"""
+    rate_limit.add_limit("basic", [Limit(2, 5)], defaults=True)
+    rate_limit.add_limit("basic", [Limit(2, 5)])
+    return rate_limit
+
+
+@pytest.fixture(scope="module")
+def commit():
+    """We need to try to commit objects during the actual test"""
+    return None
+
+
+def test_rules_exclusivity(authorization, rate_limit):
+    """Test that server will reject object with implicit and explicit defaults defined simultaneously"""
+    for component in [authorization, rate_limit]:
+        result = component._object_def_action("create", auto_raise=False)  # pylint: disable=protected-access
```

Comment:
This is very weird call I would use component.commit and pytest.raises

Reply:
I want to catch mutually exclusive error message here. Commit/create from the oc library would show only something basic like `Error during object creation`

Reply:
I believe you can still instropect the error if caught that way. In any way using protected function doesnt seem optimal and there is almost certainly a better method

Reply:
You actually were right, pytest.rises match will catch the message. Thanks